src/instructlab/configuration.py

```diff
@@ -457,6 +474,11 @@ def get_default_config() -> Config:
             deepspeed_cpu_offload_optimizer=False,
             additional_args={},
             is_padding_free=False,
+            phased_phase1_num_epochs=10,
```

Comment:
do these need upstream defaults? This might fit better in the "internal" cfg we are adding.

But this is fine for a first pass

Reply:
is there a reason upstream cant use these defaults?

Reply:
I honestly figure that we'll need to tackle the broader question of "what do we do for N different training strategies with different configurations" so I'm just hoping to have the config represented somewhere.